ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_splitting_arrays.py

```diff
@@ -0,0 +1,30 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers import array_indices_axis
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# split
+@handle_frontend_test(
+    class_tree="numpy.ndarray"
+    init_tree="numpy.array",
+    method_name="__split__",
+)
+def test_numpy_split(
+    as_variable,
+    dtype_and_x,
+    native_array,
+    num_positional_args,
+):
+    indices_or_sections, ary, axis = array_indices_axis

```

Comment:
`array_index_axis` should be inside `handle_frontend_test`, pass in the right keyword arguments for the same. Then unpack inside the test function. Look at how other functions have been implemented, and you'll get an idea !